components/sidebar-change-theme.vue

```diff
@@ -1,7 +1,50 @@
 <template>
-  <div>
-    Motyw:
-    <button @click="$colorMode.preference = 'light'">Jasny</button>
-    <button @click="$colorMode.preference = 'dark'">Ciemny</button>
+  <div class='change_theme__container'>
```

Comment:
Odkąd zobaczyłem w kodzie forum (bo to wzięte z obecnego?) ten HTML dla kontenera do zmiany motywu, to cały czas mam z tyłu głowy wątpliwość, czy tutaj struktura
```html
<div>
  <span>Motyw</span>

  <div>
    <button>Jasny</button>
    <button>Ciemny</button>
  </div>
</div>
```
jest ok... Wygląda to trochę na [`<dl>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) + [`<dt>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dt) (nagłówek) z [`<dd>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dd) (przyciski), ale to tylko jedna taka sekcja, więc na listę nie bardzo pasuje. [`<section>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) lub [`<article>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article)? Nie wiem, czy będą ok dla takiego widgetu.

IMO fajnie by było, gdyby @Comandeer się tutaj wypowiedział. Ciekaw jestem, czy moje zastrzeżenia co do obecnej struktury (`<div>`y + `<span>`) są uzasadnione.

Reply:
Div diva divem pogania. Pewnie można byłoby tu kombinować. Zamknąć cały ten component w `<section>`. "Motyw" wrzucić do `h2`. Z przycisków zrobić listę. Ale chwilowo nie będę wstrzymywał PR-ów. To można poprawić później.